src/test/java/uk/gov/justice/probation/courtcasematcher/service/TelemetryServiceTest.java

```diff
@@ -102,12 +103,27 @@ static void beforeEach() {
 
     @DisplayName("Simple record of event with no properties")
     @Test
-    void whenMessageReceived_thenRecord() {
+    void whenEvent_thenRecord() {
         telemetryService.trackEvent(TelemetryEventType.COURT_LIST_RECEIVED);
 
         verify(telemetryClient).trackEvent("PiCCourtListReceived");
     }
 
+
+    @DisplayName("Record the event when an sqs message event happens")
+    @Test
+    void whenMessageReceived_thenRecord() {
```

Comment:
Looks good but also add a test for when the message ID is null, perhaps, because it is covered in the code.